x-pack/plugins/apm/server/feature.ts

```diff
@@ -20,53 +21,29 @@ export const APM_FEATURE = {
   privileges: {
     all: {
       app: ['apm', 'kibana'],
-      api: [
-        'apm',
-        'apm_write',
-        'actions-read',
-        'actions-all',
-        'alerting-read',
-        'alerting-all',
-      ],
+      api: ['apm', 'apm_write'],
       catalogue: ['apm'],
       savedObject: {
-        all: ['alert', 'action', 'action_task_params'],
+        all: [],
         read: [],
       },
-      ui: [
-        'show',
-        'save',
-        'alerting:show',
-        'actions:show',
-        'alerting:save',
-        'actions:save',
-        'alerting:delete',
-        'actions:delete',
-      ],
+      alerting: {
+        all: Object.values(AlertType),
+      },
+      ui: ['show', 'save', 'alerting:show', 'alerting:save'],
```

Comment:
Even though Alerting no longer needs `alerting:save` the **apm** plugin uses it internally to detect users who  they want to show the Alerting flyout to, so I haven't removed it here.